### ISSUE TITLE
Impossible to view a whole trace's graph at once

### ISSUE THREAD

**Issue by dominikh** [sourcegraph/appdash#168]
When looking at a trace (`/traces/<id>`), the graph only shows two spans (the root, and one child). Clicking on the child loads a new page that shows the child and its child. There is no way to view the entire trace's graph at once. Screenshot with example attached:

![2016-05-08-131156_1380x802_scrot](https://cloud.githubusercontent.com/assets/39825/15097537/7bbbf12a-151e-11e6-8cc8-ce44aa441260.png)

**Comment by dominikh** [comment]
Worse, the current design if the UI doesn't allow for displaying the entire tree at once, because the tree structure isn't visible. If each span had more than one child, one couldn't tell which span belongs to which.

**Comment by slimsag** [comment]
Some clarification: Viewing a trace `/traces/<id>` shows the root span (`host1` above) and _all of its children_. I can confirm this behavior as working with `go run ./examples/cmd/webapp-influxdb/main.go` which shows:

![image](https://cloud.githubusercontent.com/assets/3173176/15132184/b6e45a6a-160b-11e6-88ac-0ec147a27aea.png)

In your case, the root span `host1` appears to have only one child span `host2`. Note that `profile view` shows _all spans in the trace_, so _these two views should differ_.


In response to the feature request of viewing all spans in a trace at once via this view, I agree it's a good idea.

**Comment by dominikh** [comment]
Yes, this is a feature request, not a bug report. I'm sorry if I wasn't clear enough about that.

**Comment by slimsag** [comment]
No trouble at all, my misunderstanding, and thank you for raising this issue 😄

**Comment by solher** [comment]
What would be the current good practice? Have one root span and flatten the span tree in multiple children or wait for the feature to be implemented and keep a full span tree?
